src/frontends/tensorflow/tests/convert_tricky_models.cpp

```diff
@@ -550,6 +550,11 @@ TEST_F(FrontEndConversionWithReferenceTestsF, MetaGraphCutIdentity) {
     }
 }
 
+TEST_F(FrontEndConversionWithReferenceTestsF, MetaGraphMMAPCompare) {
+    { model = convert_model("metagraph_variables/graph.meta"); }
+    { model_ref = convert_model("metagraph_variables/graph.meta", nullptr, {}, {}, {}, {}, {}, true); }
+}
```

Comment:
we look to test both cases without mmap. Can we have a test with enabled mmap?
disable_mmap = true and by default it is also `true`